src/libraries/System.Security.Cryptography.Algorithms/src/Internal/Cryptography/Helpers.cs

```diff
@@ -84,5 +84,39 @@ internal static void ConvertIntToByteArray(uint value, byte[] dest)
             dest[2] = (byte)((value & 0xFF00) >> 8);
             dest[3] = (byte)(value & 0xFF);
         }
+
+        public static bool TryGetSizeInBytes(this HashAlgorithmName hashName, out int size)
+        {
+            if (hashName.Name == HashAlgorithmName.SHA256.Name)
```

Comment:
Why not `if (hashName == HashAlgorithmName.SHA256)`?

Reply:
I think I initially wanted to first ref equals and after that string comparison but then simplified it since it's cheap comparing to operation itself, will change